common/src/tflite.cc

```diff
@@ -246,12 +258,13 @@ void tflite_classify() {
   if (kTfLiteOk != interpreter->Invoke()) {
     puts("Invoke failed.");
   }
+  printf("\n");
   uint32_t end = perf_get_mcycle();
 #ifndef NPROFILE
-  profiler->Log();
-#endif
+  profiler->LogCsv();
```

Comment:
I sent this one last night in https://github.com/google/CFU-Playground/pull/158 :-)